students/src/main/java/com/example/students/controller/StudentController.java

```diff
@@ -14,5 +14,37 @@
 @RestController
 @RequiredArgsConstructor
 public class StudentController {
+    private final StudentService studentService;
 
+    @GetMapping("/student/{id}/name/full")
```

Comment:
it means that you're getting from name resource some "full" field

Reply:
yes i try to use that pattern:
|
| -- name : |--first
                   |--second
                   |--last
                   |--full

Reply:
updated